torch/_prims/__init__.py

```diff
@@ -1879,7 +1879,8 @@ def _reshape_aten(a: Tensor, shape: ShapeType) -> Tensor:
 
 def _rev_meta(a: TensorLikeType, dims: DimsSequenceType) -> TensorLikeType:
     utils.validate_dimension_indices(a.ndim, dims)
-    return TensorMeta(a)
+    out = torch.empty_like(a, memory_format=torch.preserve_format)
```

Comment:
Might as well add a comment for why this call to `empty_like` is necessary

Reply:
Will add as a tip for writing proper meta function. 
empty_like preserves meta fields, including size, stride, device, memory_layout...